examples/pytorch/speech-recognition/run_speech_recognition_seq2seq.py

```diff
@@ -85,15 +86,24 @@ class ModelArguments:
         default="main",
         metadata={"help": "The specific model version to use (can be a branch name, tag name or commit id)."},
     )
-    use_auth_token: bool = field(
-        default=False,
+    token: bool = field(
+        default=None,
```

Comment:
Have to be `None` in order to check if it is set or not

Reply:
Type should be `str`.